docs/apache-airflow-providers/core-extensions/notifications.rst

```diff
@@ -0,0 +1,26 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Notifications
+-------------
+This is a summary of all Apache Airflow Community provided implementations of notifications.
+
+Notifications allows you to send messages to external systems when a task instance/DAG changes state.
```

Comment:
```suggestion
Notifications allow you to send messages to external systems when a task instance/DAG changes state.
```

Should “DAG” be “DAG run” instead?

Also I wonder if this should contain a reference to somewhere (to howto page?)